rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -464,9 +458,7 @@ struct Controller {
     ctlr_tx: CtlrTx,
     ipc_client: ipc::Client,
     log_filter_reloader: logging::Reloader,
-    /// Tells us when to wake up and look for a new resource list. Tokio docs say that memory reads and writes are synchronized when notifying, so we don't need an extra mutex on the resources.
-    notify_controller: Arc<Notify>,
-    resources: Arc<ArcSwap<Vec<ResourceDescription>>>,
```

Comment:
This Arc and Notify stuff was all overkill from the start, so I'm tossing it now